includes/admin/class-wc-rest-payments-reader-controller.php

```diff
@@ -296,4 +321,47 @@ function ( bool $served, WP_HTTP_Response $response ) : bool {
 			[ 'Content-Type' => 'text/html; charset=UTF-8' ]
 		);
 	}
+	/**
+	 * Returns HTML to preview a print receipt
+	 *
+	 * @param WP_REST_Request $request Full data about the request.
+	 * @return WP_HTTP_Response|WP_Error
+	 * @throws \RuntimeException Error collecting data.
+	 */
+	public function preview_print_receipt( WP_REST_Request $request ) {
+		$sample_order = new WC_Payments_Printed_Receipt_Sample_Order();
```

Comment:
We can inline the value, so the local variable can be dropped.

Reply:
Done!